cirq/google/common_serializers_test.py

```diff
@@ -607,3 +607,64 @@ def test_wait_gate():
     op = cirq.WaitGate(cirq.Duration(nanos=20)).on(q)
     assert gate_set.serialize_op(op) == proto
     assert gate_set.deserialize_op(proto) == op
+
+
+@pytest.mark.parametrize(('gate', 'theta', 'phi'), [
+    (cirq.ISWAP**0.5, -np.pi / 4, 0),
+    (cirq.ISWAP**-0.5, np.pi / 4, 0),
```

Comment:
Can you explain to me how does it work, since you didn't specify serializer for ISwapPowGate with exponent equal to -0.5?

Reply:
The serializer checks for abs(exponent)==0.5 (mod 2pi)